schema/slots/float-slot.json

```diff
@@ -0,0 +1,17 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Float Slot",
+    "description": "Float Slot",
+    "properties": {
+        "t": {
```

Comment:
It might be helpful to on-top of having a "t" field for data type, to also include something like "c" for context.

This does introduce some questions:
How granular are we with context? There are many different types of Float Slots (opacity, rotation, blurriness, sharpness, etc.)
If someone wants to group two float slots of the same "t" but different "c", how do we represent the slot group?

These questions aren't trivial, but thankfully can exist along side the current proposal, so we can revisit this in a follow up proposal/extension.